unit-tests/algo/d2rgb/test-all-set-option.py

```diff
@@ -0,0 +1,94 @@
+import pyrealsense2 as rs, test
+import time
+
+#L515
+
+dev = test.find_first_device_or_exit()
+depth_sensor = dev.first_depth_sensor()
+color_sensor = dev.first_color_sensor()
+
+previous_depth_frame_number = -1
+previous_color_frame_number = -1
+after_set_option = 0
+
+def collect_frames_after_set(): 
+    global after_set_option
+    after_set_option = 1
```

Comment:
But we're already after the set-option here... what about the frames that arrived *during* the set-option??